README.md

```diff
@@ -1,2 +1,5 @@
 # git-demo
 to demonstrate how to use git and GitHub
+
+Name: Emma Vejcik
+GitHub Link: https://github.com/evejcik
```

Comment:
Could you please add a `-` so this becomes a list item?

Also, please make your name into a link that goes to that URL. The URL should not be visible.